pkgs/applications/window-managers/waybox/default.nix

```
@@ -16,13 +16,13 @@
 
 stdenv.mkDerivation rec {
   pname = "waybox";
-  version = "unstable-2020-05-01";
+  version = "unstable-2021-04-07";
 
   src = fetchFromGitHub {
     owner = "wizbright";
     repo = pname;
-    rev = "93811898f0eea3035145f593938d49d5af759b46";
-    sha256 = "IOdKOqAQD87Rs3td8NVEgMnRF6kQSuQ64UVqeVqMBSM=";
+    rev = "309ccd2faf08079e698104b19eff32b3a255b947";
+    sha256 = "0kzss266jf1kgwnhk42vaakbmn269nphlq9xjsapp6mhccd9qz8v";
```

can you use the SRI format?

```suggestion
    hash = "sha256-G32cGmOwmnuVlj1hCq9NRti6plJbkAktfzM4aYzQ+k8=";
```

---

Sure. Feel free to apply such "cosmetic" changes via the GH UI and use squash and merge (or manually via Git) ;)

But may I ask what your reasons for that are? `nix-prefetch-url` still uses the old format (at least with stable hash) and AFAIK `sha256 = "sha256-...` is still preferable for `nix-update` as that cannot handle `hash = ...` yet (but maybe that has already changed). So yes, we're moving towards SRI and there's nothing wrong with that but AFAIK the old hashes are still perfectly fine (or am I missing something?).

---

Uniformization. 
I am using the `nix` tool for building and testing the packages, and they output the hashes in SRI format. I am not using nix-prefetch-url at all, I just update the source and use `sha256="";` (an empty string). It returns me the expected hash, always in SRI format.

Because of it, I am using SRI hashes. Indeed, there are some corners to be trimmed (e.g. `hash` argument fails for `fetchsvn`), but at least the `sha256=<SRI hash format>;` always worked as expected.